File: TRNMNT.Core/Model/Round/RoundResultModel.cs
using System;
using TRNMNT.Core.Enum;

namespace TRNMNT.Core.Model.Round
{
    public class RoundResultModel
    {
        public Guid RoundId { get; set; }
        public Guid WinnerParticipantId { get; set; }
        public int CompleteTime { get; set; }
        public MatchResultTypeEnum RoundResultType { get; set; }
        public SubmissionTypeEnum SubmissionType { get; set; }
        public int AParticipantPoints { get; set; }
        public int AParticipantAdvantages { get; set; }
        public int AParticipantPenalties { get; set; }
        public int SecondParticipantPoints { get; set; }
        public int SecondParticipantAdvantages { get; set; }
        public int SecondParticipantPenalties { get; set; }
    }
}

File: TRNMNT/ClientApp/app/core/model/round-result.model.ts
import { ParticipantModelBase } from './participant.models';

export class RoundResultModel {
    roundId: AAGUID;
    winnerParticipantId: AAGUID;
    AParticipantPoints: number;
    AParticipantAdvantages: number;
    AParticipantPenalties: number;
    BParticipantPoints: number;
    BParticipantAdvantages: number;
    BParticipantPenalties: number;
    submissionType : number;
    roundResultType : number;
    completeTime: number;
}